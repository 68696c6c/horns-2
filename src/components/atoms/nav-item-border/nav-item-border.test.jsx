/* global describe, it, expect */
import React from 'react'

import { render } from 'setup-test'

import NavItemBorder from '.'

describe('NavItemBorder', () => {
  it('should render as default', () => {
    const { container } = render(<NavItemBorder>example</NavItemBorder>)
    expect(container).toMatchSnapshot()
  })
})
