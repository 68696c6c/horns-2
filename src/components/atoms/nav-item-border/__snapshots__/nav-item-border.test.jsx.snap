// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavItemBorder should render as default 1`] = `
.emotion-0 {
  background: rgb(255,255,255);
  color: rgb(1,1,1);
  border-color: rgb(216,216,216);
  cursor: pointer;
  font-family: Helvetica,sans-serif;
  font-size: medium;
  font-style: normal;
  font-weight: 400;
  line-height: 1em;
  -webkit-text-decoration-line: none;
  text-decoration-line: none;
  padding-top: 16px;
  padding-bottom: 16px;
  padding-left: 16px;
  padding-right: 16px;
  display: inline-block;
}

.emotion-0:disabled,
.emotion-0.inactive {
  cursor: pointer;
  background: rgb(191,191,191);
  color: rgb(1,1,1);
  border-color: rgb(165,165,165);
  -webkit-text-decoration-line: none;
  text-decoration-line: none;
}

.emotion-0:not(:disabled):not(.inactive):hover {
  background: rgb(235,235,235);
  color: rgb(1,1,1);
  border-color: rgb(216,216,216);
  -webkit-text-decoration-line: none;
  text-decoration-line: none;
}

.emotion-0:not(:disabled):not(.inactive):active {
  background: rgb(216,216,216);
  color: rgb(1,1,1);
  border-color: rgb(235,235,235);
  -webkit-text-decoration-line: none;
  text-decoration-line: none;
}

<div>
  <a
    class="emotion-0"
    color="background"
    cursor="pointer"
    href="#"
  >
    example
  </a>
</div>
`;
